examples/module-federation/mf-react-component/package.json

```diff
@@ -13,17 +13,22 @@
   "types": "./dist/cjs/index.d.ts",
   "scripts": {
     "build": "rslib build",
-    "serve": "pnpm build && http-server -p 3001 ./dist/ --cors"
+    "dev": "rslib dev:mf",
+    "serve": "pnpm build & http-server -p 3001 ./dist/ --cors",
```

Comment:
what does server do here?

Reply:
`serve` command is to test assets of mf format is correct, e2e test will use this command https://github.com/web-infra-dev/rslib/blob/main/tests/playwright.config.ts#L17